### ISSUE TITLE
Sertoli cell-only phenotype

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7578]
**Preferred term label:**
Sertoli cell-only phenotype

**Synonyms**

**Definition (free text, please give PubMed ID)**
A histological phenotype observed on testicular biopsy in which only Sertoli cells line the seminiferous tubules of the testis.

Comment: Sertoli cells are found in the convoluted seminiferous tubules and form part of the blood-testis barrier in males. With Sertoli cell-only phenotype, spermatogenic cells, which are normally located in between the Seroli cells, are absent.
PMID:30484998

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
619831 
PMID: 32719396   2/2
also  305700 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
